virt/release_notes/virt-4-21-release-notes.adoc

```diff
@@ -76,6 +76,14 @@ The SVVP certification applies to:
 [id="virt-4-21-deprecated_{context}"]
 == Deprecated features
 
+The `HotplugVolume` feature gate is deprecated::
+
+The `HotplugVolume` feature gate, which allows you to add storage without restarting your VM, is deprecated and will be removed in a future release. This feature gate will be replaced by `DeclarativeHotplugVolumes`.
+
+NOTE: `DeclarativeHotplugVolumes` does not support ephemeral hotplug volumes. Ephemeral hotplug volumes are volumes that are hotplugged to a VMI and do not persist in the owner VM. Existing ephemeral hotplug volumes are automatically detached after you switch to `DeclarativeHotplugVolumes`.
+
+link:https://issues.redhat.com/browse/CNV-73301[CNV-73301]
```

Comment:
This Jira is restricted to Red Hat employees and Contractors only, so the build check is throwing an error. Can you change the security level or point to a Jira that is public?